graphql/src/values/coercion.rs

```diff
@@ -42,6 +42,10 @@ impl MaybeCoercible<ScalarType> for q::Value {
                     Err(q::Value::Int(num))
                 }
             }
+            ("Int8", q::Value::Int(num)) => {
+                let n = num.as_i64().ok_or_else(|| q::Value::Int(num.clone()))?;
+                Ok(r::Value::Int(n))
+            }
```

Comment:
I didn't realize that the various values were already using `i64` internally .. wild. We should probably have a ticket to narrow `r::Value::Int` to `i32` at some point.

Reply:
Yeah, I wasn't sure if that's the right time to touch this or now 😆 I created a ticket here: https://github.com/graphprotocol/graph-node/issues/4552